### PR TITLE
Update contributors.html

### PR DESCRIPTION
**Pull request by Hiphop602**
## PR Summary

Fixed a minor bug in navbar

### CODE REVIEW

**Review by StarManTheGamer — CHANGES_REQUESTED** [review]
You're having the page go to "index", which won't work on local versions of the documents. Instead, please put "/" so it works on local and live version.

**Comment by StarManTheGamer** [comment]
Also please do the same change for the index page, as the same page's button only goes to "#" and not "/"

**Comment by Hiphop602** [comment]
See, for local /  is not working only filename.html is. Should I do that instead?

**Comment by StarManTheGamer** [comment]
No - I'm not sure how yours is setup, but it should be that when you put / it'll take you to the IP address you're hosting mkdocs on, without needing /index or /index.html

**Comment by Hiphop602** [comment]
Welp, when I'm using the live version only the filename.html path is working. Can you check once?

**Comment by StarManTheGamer** [comment]
Yeah, i can show you what i mean tomorrow. I'm getting ready for bed and already put away my laptop

**Comment by Hiphop602** [comment]
Ohk

**Comment by StarManTheGamer** [comment]
Your latest commit broke the main page. This needs to be fixed before I'm okay with merging.
image
image

File: docs/index.html [hunk 1]
```diff
@@ -34,6 +34,32 @@
 			background-repeat: no-repeat;
 		"
 	>
+
+		<!-- Script for scrolling-->
+		<script>		
+
+			window.addEventListener('scroll', function() {
```
Comment (Alyxsqrd) [inline docs/index.html:41]: Highly inefficient code... searching for element IDs and calculating bounds _everytime_ the page is scrolled.

**Review by Alyxsqrd — CHANGES_REQUESTED** [review]

**Comment by Hiphop602** [comment]
> Your latest commit broke the main page. This needs to be fixed before I'm okay with merging. image image

Thanks for pointing that out! I'll fix that once I get free.



> Your latest commit broke the main page. This needs to be fixed before I'm okay with merging. image image

Thanks for pointing that out! I'll fix it once I get free. 

-**Hiphop602**

**Comment by Hiphop602** [comment]
> Your latest commit broke the main page. This needs to be fixed before I'm okay with merging. image image

Thanks for pointing that out!! I'll fix it once I get free.

**Comment by StarManTheGamer** [comment]
> > Your latest commit broke the main page. This needs to be fixed before I'm okay with merging. image image
> 
> Thanks for pointing that out!! I'll fix it once I get free.

You'll also have to change what Alyxsqrd suggested, he has priority over me. I'm not an expert with javascript so I didn't know that it was inefficient

File: docs/index.html (hunk 1 above)
Comment (IndexingGitHub) [reply docs/index.html:41]: You are doing a .getElementByID **every** single time the page scrolls, pretty sure thats what they are referencing.

**Comment by Hiphop602** [comment]
Hello, can you guys check out the webpage now?

> https://hiphop602.github.io/Docs/docs/index.html

**Comment by Hiphop602** [comment]
Just noticed that the footer is slightly aligned to the left.... I'll try fixing that as well.

**Comment by IndexingGitHub** [comment]
> Just noticed that the footer is slightly aligned to the left.... I'll try fixing that as well.  

This has already been noticed on mobile. I just haven't fixed it yet, if you would like to fix it then go ahead.

**Comment by Hiphop602** [comment]
> > Just noticed that the footer is slightly aligned to the left.... I'll try fixing that as well.
> 
> This has already been noticed on mobile. I just haven't fixed it yet, if you would like to fix it then go ahead.

Oh... have you guys used bootstrap or something?

**Comment by IndexingGitHub** [comment]
> > > Just noticed that the footer is slightly aligned to the left.... I'll try fixing that as well.
> > 
> > 
> > This has already been noticed on mobile. I just haven't fixed it yet, if you would like to fix it then go ahead.
> 
> Oh... have you guys used bootstrap or something?

I made the homepage frontend, yes, it is using Bootstrap. I did not use the default Bootstrap footer however. It's just a row with columns (I didn't test mobile compatibility much as it doesn't matter much as long as the page is readable and looks good - and that the documentation is good).

**Pull request closed without merging**